ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -6,22 +6,26 @@ import { Button } from 'components/forms/Button'
 import { QuestionDesigner } from './QuestionDesigner'
 import { TextInput } from 'components/forms/TextInput'
 import { baseQuestions } from './questions/questionTypes'
-import _ from 'lodash'
+import _, { isEmpty } from 'lodash'
```

Comment:
I believe we also have tree shaking that means that the import style should have no impact on bundle size.  In general, I prefer specific imports since it makes the dependencies clearer at the top of the file, but it's not a strong preference.